src/utils/default-drives-manager.ts

```diff
@@ -76,7 +77,8 @@ export class DefaultDrivesManager implements IDefaultDrivesManager {
 
     private async preserveDrivesById(
         drivesIdsToRemove: string[],
-        drives: string[]
+        drives: string[],
+        detach = false
```

Comment:
Thinking from an external dev perspective, I'd expect `preserveDrivesById` to receive `driveIdsToPreserve` instead of the ids to remove. The third argument could be removeStrategy = "delete" | "detach".
What do you think? Would it make calling this method harder?